### PR TITLE
docs: update 13.faq

### PR DESCRIPTION
**Pull request by fenghuazzm**
Add some issues

### CODE REVIEW

File: documentation20/en/13.faq/docs.md [hunk 1]
```diff
@@ -159,3 +159,22 @@ ALTER LOCAL RESETLOG;
 
 This means wiping up all client-generated log files on the machine.
 
+## 18. How to solve the golang component compilation failure?
+
+The new version of TDengine 2.3.0.0 contains a taosAdapter standalone component developed in go, which needs to be run separately, replacing the httpd built into taosd. Provides data access functionality including the original httpd functionality and support for a variety of other software (Prometheus, Telegraf, CollectD, StatsD, etc.).
```
Comment (gccgdb1234) [inline documentation20/en/13.faq/docs.md]: The new version of TDengine 2.3.0.0 -> From version 2.3.0.0
这句话最初加的时候是正确的，但随着后续版本的增加，已经是病句了

Comment (gccgdb1234) [reply documentation20/en/13.faq/docs.md]: From version 2.3.0.0, a new component `taosAdapter` developed in go has been added in TDengine. `taosAdapter` needs to be run separately to replace the httpd built in `taosd`. It provideds REST API and data flow a variety of 3rd party tools like Prometheus, Telegraf, CollectD, StatsD, etc, can be inserted into TDengine through it.

File: documentation20/en/13.faq/docs.md [hunk 2]
```diff
@@ -159,3 +159,22 @@ ALTER LOCAL RESETLOG;
...
+
+To compile with the latest Develop branch, run `git submodule update --init --recursive` to download the taosAdapter repository code and then compile it.
```
Comment (gccgdb1234) [inline documentation20/en/13.faq/docs.md]: 这句话可以去掉， 在taosAdapter的手册中已经有详细说明了。 加一个Reference即可
For details about `taosAdapter`, please refer to [taosAdapter](/reference/taosAdapter).

File: documentation20/en/13.faq/docs.md [hunk 3]
```diff
@@ -159,3 +159,22 @@ ALTER LOCAL RESETLOG;
...
+
+## 19. How do I query the disk space occupied by data?
+
+By default, TDengine data files are stored in /var/lib/taos and log files are stored in /var/log/taos.
+
+To check the size of all data files, run the following command: `du -sh /var/lib/taos/vnode --exclude='wal'` .The wal directory is excluded here because the size is almost fixed in the case of continuous heavy writes, and wal directories are emptied whenever TDengine is normally shut down.
```
Comment (gccgdb1234) [inline documentation20/en/13.faq/docs.md]: in the case of -> in case of

Comment (gccgdb1234) [inline documentation20/en/13.faq/docs.md]: 这个命令是Shell命令，要说明一下
run command `xxxxx` on Linux Shell.

File: documentation20/en/13.faq/docs.md [hunk 4]
```diff
@@ -159,3 +159,22 @@ ALTER LOCAL RESETLOG;
...
+
+To check the size of a single database, run the `show vgroups;` command in the taos command to specify a database, go to /var/lib/taos/vnode with the obtained VGroup ID to check the contained folder size.
+
+If you only want to see the block distribution and size of a (super) table, You can use _block_dist functions: 
+
+```mysql
+SELECT _block_dist () the FROM {tb_name | stb_name}
```
Comment (gccgdb1234) [inline documentation20/en/13.faq/docs.md]: 多了个 "the" ? SQL语句里不应该有 the

File: documentation20/en/13.faq/docs.md [hunk 5]
```diff
@@ -159,3 +159,22 @@ ALTER LOCAL RESETLOG;
...
+
+To compile with the latest Develop branch, run `git submodule update --init --recursive` to download the taosAdapter repository code and then compile it.
+
+## 19. How do I query the disk space occupied by data?
+
+By default, TDengine data files are stored in /var/lib/taos and log files are stored in /var/log/taos.
+
+To check the size of all data files, run the following command: `du -sh /var/lib/taos/vnode --exclude='wal'` .The wal directory is excluded here because the size is almost fixed in the case of continuous heavy writes, and wal directories are emptied whenever TDengine is normally shut down.
+
+To check the size of a single database, run the `show vgroups;` command in the taos command to specify a database, go to /var/lib/taos/vnode with the obtained VGroup ID to check the contained folder size.
```
Comment (gccgdb1234) [inline documentation20/en/13.faq/docs.md:174]: To check the size of a single database, firstly run command `show vgroups <dbname>`  for a specific database in TDengine CLI `taos`  to get the list of vgroups for the database, then check the folder size corresponding to the vgroups under /var/lib/taos/vnode directory.

File: documentation20/en/13.faq/docs.md (hunk 4 above)
Comment (fenghuazzm) [reply documentation20/en/13.faq/docs.md]: 咦...手滑了

**Pull request merged**